index.html

```diff
@@ -2,90 +2,120 @@
-          <a href="https://github.com/jlcarmic/green-lantern" target="_blank">
-            <button class="button">Green Lantern</button>
+          <p>An Alexa Skill to get information of a movie plot.</p>
+          <p>Built using NodeJS and AWS Lambda.</p>
+          <p>Utilizes imdb API.</p>
+          <a href="https://github.com/sonukhot/alexa-movieplot" target="_blank">
+            <button class="button">Movie Plot</button>
           </a>
         </div>
         <div class="screenshot">
-          <img src="images/greenlantern.png" />
+          <img src="images/movie.jpg" />
         </div>
       </div>
 
       <div class="project">
         <div class="screenshot">
-          <img src="./images/producthuntalexa.png" />
+          <img src="./images/weather.jpg" width="500" height="350" />
         </div>
         <div class="explanation">
-          <p>An Alexa Skill for hearing Today's Featured products.</p>
+          <p>An Alexa Skill to know Today's Weather conditions.</p>
           <p>Built using NodeJS and AWS Lambda.</p>
-          <p>Utilizes ProductHunt's API.</p>
+          <p>Utilizes Open Weather API.</p>
           </p>
-          <a href="https://github.com/jlcarmic/producthunt_alexa_skill" target="_blank">
-            <button class="button">Today's Featured</button>
+          <a href="https://github.com/sonukhot/todays-weather" target="_blank">
+            <button class="button">Today's Weather</button>
           </a>
         </div>
       </div>
 
       <div class="project">
         <div class="explanation">
-          <p>An NPM package for accessing sports<br />stats via the Stattleship API.</p>
-          <p>Built in NodeJS and published to NPM<br />as an open source package.</p>
-          <a href="https://github.com/jlcarmic/node-stattleship" target="_blank">
-            <button class="button">Node Stattleship</button>
+          <p>Member of Ashland Community Garden.</p>
+          <p>Involved in mentoring new gardeners and organize workshops for kids and adults.</p>
+          <a href="https://www.facebook.com/acgstonepark/" target="_blank">
+            <button class="button">Volunteer</button>
           </a>
         </div>
         <div class="screenshot">
-          <img class="stattleship" src="./images/stattleship.svg" />
+          <img class="stattleship" src="./images/volunteer.jpg" width="300" height="250" />
+        </div>
+      </div>
+      <div class="project">
+        <div class="screenshot">
+          <img src="./images/2ND FLOOR.jpg" width="400" height="400" />
+        </div>
+        <div class="explanation">
+          <p>A Interior Space planning and designing project.</p>
+          <p>Designed from conceptual planning, design, and execution to completion.</p>
+          <p>Coordinated with different local and remote consultants and vendors .</p>
+          </p>
+          <a>
+            <button class="button">TCS Development Center, Madhapur</button>
+          </a>
```

Comment:
I like the inclusion of these, they show some personality and help show off your other skills beyond development.